frontend/src/app/components/crm/analytics/analytics.component.ts

```diff
@@ -1,20 +1,26 @@
-import { Component, OnInit } from '@angular/core';
+import { Component, OnInit, ViewChild, AfterViewInit } from '@angular/core';
+import { MapComponent } from './map/map.component';
+import { ChartsComponent } from './charts/charts.component';
 import { Title } from '@angular/platform-browser';
 
 @Component({
   selector: 'app-analytics',
   templateUrl: './analytics.component.html',
   styleUrls: ['./analytics.component.css']
 })
-export class AnalyticsComponent implements OnInit {
+/// <reference types="googlemaps" />
+export class AnalyticsComponent implements AfterViewInit {
   chartsTabLabel = $localize`Charts`;
   mapTabLabel = $localize`Map`;
 
+  @ViewChild('map') map: MapComponent;
+  @ViewChild('charts') chart: ChartsComponent;
+
   constructor(private titleService: Title) {
     this.titleService.setTitle($localize`Analytics`);
   }
 
-  ngOnInit(): void {
+  ngAfterViewInit(): void {
```

Comment:
This should be `ngOnInit` otherwise compiler will throw an error saying you didn't implement `OnInit`